caput/tod.py

```diff
@@ -118,7 +120,7 @@ def __init__(self, files):
 
         # Fetch all meta data.
         time = np.copy(data_empty.time)
-        first_file, toclose = memh5.get_h5py_File(files[0])
```

Comment:
We went from a default of `HDF5` to a default of "guessing from filename". This will be fine if we use a consistent suffix with our hdf5 files.

Reply:
If a guess is not conclusive, maybe hdf5?